x/genutil/client/cli/gentx.go

```diff
@@ -163,12 +153,17 @@ $ %s gentx my-key-name --home=/path/to/home/dir --keyring-backend=os --chain-id=
 			}
 
 			// read the transaction
-			stdTx, err := readUnsignedGenTxFile(cdc, w)
+			stdTx, err := readUnsignedGenTxFile(clientCtx, w)
 			if err != nil {
 				return errors.Wrap(err, "failed to read unsigned gen tx file")
 			}
 
 			// sign the transaction and write it to the output file
+			txBuilder, err := clientCtx.TxConfig.WrapTxBuilder(stdTx)
+			if err != nil {
+				return fmt.Errorf("error creating tx builder")
```

Comment:
Please be careful of situations like this ;-)

```suggestion
				return fmt.Errorf("error creating tx builder: %w", err)
```

Reply:
Wow, good eye! Thanks!